swagger/swagger.yml

```diff
@@ -68,7 +68,7 @@ paths:
           in: query
           type: integer
           format: int32
```

Comment:
if you remove this format from here
you don't need to typecast in code.

Reply:
Done only in case of limit query parameter in get request, kindly check.